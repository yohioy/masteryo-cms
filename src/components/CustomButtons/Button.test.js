import * as React from 'react';
import '@testing-library/jest-dom/extend-expect';
import { cleanup, render } from '@testing-library/react';

import RegularButton from './Button';

afterEach(cleanup);

test('has correct links', () => {
    const { container, getByText, queryByTestId, getByTestId } = render(<RegularButton />);

    expect(getByTestId('button-component')).toBeTruthy();
    expect(container).toMatchSnapshot();
});
