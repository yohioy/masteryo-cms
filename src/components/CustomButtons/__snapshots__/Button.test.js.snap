// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`has correct links 1`] = `
<div>
  <button
    class="MuiButtonBase-root MuiButton-root makeStyles-button-1 MuiButton-text"
    data-testid="button-component"
    tabindex="0"
    type="button"
  >
    <span
      class="MuiButton-label"
    />
    <span
      class="MuiTouchRipple-root"
    />
  </button>
</div>
`;
